applications/StructuralMechanicsApplication/tests/test_harmonic_analysis.py

```diff
@@ -31,27 +30,25 @@ def _add_variables(self,mp):
         mp.AddNodalSolutionStepVariable(StructuralMechanicsApplication.POINT_LOAD)
 
     def _solve_eigen(self,mp,echo=0):
-        feast_system_solver_settings = KratosMultiphysics.Parameters("""{ }""")
-
-        eigensolver_settings = KratosMultiphysics.Parameters("""
-        {
-                "perform_stochastic_estimate": false,
-                "lambda_min": 1.0e-2,
-                "lambda_max": 25.0,
-                "search_dimension": 7
-        }
-        """)
-        if not (hasattr(KratosMultiphysics.ExternalSolversApplication,"PastixComplexSolver")):
-            self.skipTest('"PastixComplexSolver" not available')
-
-        feast_system_solver = ExternalSolversApplication.PastixComplexSolver(feast_system_solver_settings)
-        eigen_solver = ExternalSolversApplication.FEASTSolver(eigensolver_settings, feast_system_solver)
+        eigensolver_settings = KratosMultiphysics.Parameters("""{
+            "solver_type": "eigen_feast",
```

Comment:
I think you are missing the skiptest here if the EigenSolversApp is not there / if it is not compiled with feast
=> pretty sure this will fail in the CI like this

Reply:
The test is only added to the suite, if FEAST is available in the first place (see `test_StructuralMechancisApplication.py`, l. 427)

Reply:
hm ok
I think it would be cleaner to do it directly in the tests, but I am anyway planning to refactor these tests a bit in the future, so for now it can stay as is